microsetta_interface/implementation.py

```diff
@@ -106,8 +106,16 @@ class Source:
     5: 'Questions about your interest in the microbiome',
     6: 'Questions specific to COVID19',
     VIOSCREEN_ID: 'Our standard food frequency questionnaire',
-    MYFOODREPO_ID: 'The MyFoodRepo picture-based, artificial '
-                   'intelligence backed, diet assessment platform'
+    MYFOODREPO_ID: 'US Participants only: By sharing photos of the food you '
```

Comment:
The instrument is fit for US participants but it is not limited to them

Reply:
Are we trying to discourage international participants from using MFR, but allowing them to if they so desire? Or do we welcome anyone and everyone to use it?